qtp_sequencing/tests/test_validate.py

```diff
@@ -115,7 +129,7 @@ def test_validate_multiple_error(self):
                      "1.SKB3.640195": {"run_prefix": "prefix2"}}
         files = {'Unknown': ['/path/to/file1.fastq']}
         atype = "FASTQ"
-        job_id = self._create_template_and_job(prep_info, files, atype)
+        job_id, _ = self._create_template_and_job(prep_info, files, atype)
```

Comment:
@antgonza Don't you think that parameters should be checked as well, now that they're being returned? It seems like this function is only getting half-tested.

Reply:
Do you mean `_create_template_and_job`? If yes, that's a test help method to avoid duplication of code; thus not really needed to be tested; in other words, we normally don't write tests for the tests. If not, which function are you talking about?

Reply:
Ah I see. Yes, that's the one.